integration-tests/tests/syscall_exit.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the exit syscall payload test.
```

Comment:
I think the module name is self-describing. The only tests shown here are ones for `exit()` calls, so instead of saying "including ..." we might as well just shorten it to something like "sys_exit tests"

Reply:
Fixed.